todolist/settings.py

```diff
@@ -62,21 +62,16 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+        'default': {
+            'ENGINE': 'mysql.connector.django',
+            'NAME': 'app_db',
+            'USER': 'app_user',
+            'PASSWORD': '1234',
```

Comment:
Hardcoding the database password is a security risk. Consider using environment variables or a secure vault to manage this sensitive information.